### PR TITLE
Add docs for generating docs on quick start guide.

### PR DESCRIPTION
**Pull request by leonsim**
(Thanks for sending a pull request! Please make sure to read the contribution guidelines, then fill out the blanks below.)

What changes were proposed in this pull request?
------------------------------------------------
Update docs.

Does this close any currently open issues?
------------------------------------------
Nope

How was this patch tested?
--------------------------
Not yet.

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -287,6 +298,12 @@ service.
 
     !IrisClassifier info
 
+You can use `docs` command to get all APIs in OpenAPI format.
+
+.. code-block:: python
+
+    !IrisClassifier docs
```
Comment (parano) [inline docs/quickstart.rst]: could you also add the other form of the same CLI command here: ```bentoml docs {ARCHIVE_PATH}```

Comment (parano) [reply docs/quickstart.rst]: And also add the same thing to the quick start jupyter notebook?

Comment (leonsim) [reply docs/quickstart.rst]: Yes sure

**Comment by parano** [comment]
Thanks @leonsim, looks great!

**Pull request merged**